media-gfx/blender/files/blender-3.2.2-malloc_stats-musl.patch

```diff
@@ -0,0 +1,16 @@
+# Function malloc_stats is only available on GLIBC, so check that before
+# defining HAVE_MALLOC_STATS
+# Taken from void linux pathc, refer here:
+# https://github.com/void-linux/void-packages/blob/master/srcpkgs/blender/patches/0001-musl-fixes.patch
+--- a/intern/guardedalloc/intern/mallocn_intern.h
++++ b/intern/guardedalloc/intern/mallocn_intern.h
+@@ -17,8 +17,7 @@
+ #undef HAVE_MALLOC_STATS
+ #define USE_MALLOC_USABLE_SIZE /* internal, when we have malloc_usable_size() */
+
+-#if defined(__linux__) || (defined(__FreeBSD_kernel__) && !defined(__FreeBSD__)) || \
+-    defined(__GLIBC__)
++#if (defined(__FreeBSD_kernel__) && !defined(__FreeBSD__)) || defined(__GLIBC__)
```

Comment:
Instead of relying on `__GLIBC__` I could improve it to something like checking for function using cmake. But would like to hear from devs first.

Reply:
yes please (thanks for doing it already)